test/modules/standard/FileSystem/lydia/parallelDirState/chdirAllLocales.skipif

```diff
@@ -0,0 +1 @@
+CHPL_COMM == none
```

Comment:
My preference would actually be to have this also test with CHPL_COMM=none and to have a chdirAllLocales.comm-none.good file.  I.e., it would be a shame if something about a program that used Locales.chdir/cwd idiom broke on one locale since you can still refer to Locales in a CHPL_COMM=none program.  I think all that would be required would be to remove this file and add a 1-line version of the current .good file.